<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogdecharge</class>
 <widget class="QDialog" name="Dialogdecharge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1363</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>791</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::tab-bar {
   border: 1px solid gray;
}
QTabBar::tab {
  background: gray;
  color: white;
  padding: 10px;
 }

 QTabBar::tab:selected {
  background: lightgray;
 }
QTabWidget::pane { 
   border: none;
}</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="ajout">
    <attribute name="title">
     <string>Ajouter quantité</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_Ajout">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>230</y>
       <width>591</width>
       <height>441</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Ajouter Nouvelle Quantité </string>
     </property>
     <widget class="QLabel" name="label_poids">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Poids :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_poids">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid;
border-color: rgb(0,0,0);
border-radius: 25% 10%;</string>
      </property>
      <property name="placeholderText">
       <string>Exp 33</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_proven">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid;
border-color: rgb(0,0,0);
border-radius: 25% 10%;</string>
      </property>
      <property name="placeholderText">
       <string>Exp : 21,rue azou</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_proven">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Provenance:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Type">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Type :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAnnulerDech">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: grey;
</string>
      </property>
      <property name="text">
       <string>ANNULER</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEnregDech">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>360</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: grey;
</string>
      </property>
      <property name="text">
       <string>ENREGISTRER</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_type">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>250</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid;
border-color:rgb(0, 0, 0);
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
    color: rgb(220, 220, 220);
    background-color:rgb(89, 155, 255);
    margin: 0 0 0 0;</string>
      </property>
      <item>
       <property name="text">
        <string>Veuiller choisir</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Organique</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sanitaire</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Plastique</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Metal</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QLabel" name="label_dechajout">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1366</width>
       <height>768</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_dechajout</zorder>
    <zorder>groupBox_Ajout</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>supprimer/modifier quantité</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>831</width>
       <height>351</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>SUPPRIMER/MODIFIER QUANTITE</string>
     </property>
     <widget class="QLabel" name="label_ID">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID quantité</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditId">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid color rgb(170, 85, 127);
border-radius: 25% 10%;</string>
      </property>
      <property name="placeholderText">
       <string>EXP:3</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonSupprimer">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>50</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonModifer">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>50</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonValider">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>50</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: grey;
</string>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonsupmoddech">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>260</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: grey;
</string>
      </property>
      <property name="text">
       <string>SUPPRIMER/MODIFIER</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_poidssupmod">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Poids</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPoidssupmod">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid color rgb(170, 85, 127);
border-radius: 25% 10%;</string>
      </property>
      <property name="placeholderText">
       <string>number:25</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Provenancesupmod">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>130</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid color rgb(170, 85, 127);
border-radius: 25% 10%;</string>
      </property>
      <property name="placeholderText">
       <string>Entrer une adresse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_provsupmod">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>130</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Provenance</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_typesupmod">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_typesupmod">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid;
border-color:rgb(0, 0, 0);
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
    color: rgb(220, 220, 220);
    background-color:rgb(89, 155, 255);
    margin: 0 0 0 0;</string>
      </property>
      <item>
       <property name="text">
        <string>Organique</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sanitaire</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Plastique</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Metal</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QLabel" name="label_BackSupDech">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1366</width>
       <height>768</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_BackSupDech</zorder>
    <zorder>groupBox</zorder>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Afficher Historique</string>
    </attribute>
    <widget class="QLabel" name="label_backhistdech">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1366</width>
       <height>768</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>591</width>
       <height>441</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Recherche par ID</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID Quantité</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Rechquantite">
      <property name="geometry">
       <rect>
        <x>142</x>
        <y>59</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid color rgb(170, 85, 127);
border-radius: 25% 10%;</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonrechdech">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid;
border-color:rgb(255, 255, 255);
background-color: grey;
</string>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="tableViewRech">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>370</y>
       <width>561</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tableViewAfficher">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>370</y>
       <width>561</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAfficherDech">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>310</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid;
border-color:rgb(255, 255, 255);
background-color: grey;
</string>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_tripar">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>260</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;
border-color:rgb(0, 0, 0);
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
    color: rgb(220, 220, 220);
    background-color:rgb(89, 155, 255);
    margin: 0 0 0 0;</string>
     </property>
     <item>
      <property name="text">
       <string>id_quantite</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>poids</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_typetri">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>260</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;
border-color:rgb(0, 0, 0);
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
    color: rgb(220, 220, 220);
    background-color:rgb(89, 155, 255);
    margin: 0 0 0 0;</string>
     </property>
     <item>
      <property name="text">
       <string>croissant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>decroissant</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>220</y>
       <width>591</width>
       <height>441</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Afficher Historique</string>
     </property>
    </widget>
    <zorder>label_backhistdech</zorder>
    <zorder>groupBox_3</zorder>
    <zorder>pushButtonAfficherDech</zorder>
    <zorder>comboBox_typetri</zorder>
    <zorder>comboBox_tripar</zorder>
    <zorder>tableViewAfficher</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>tableViewRech</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
